doc/sphinx/electrostatics.rst

```diff
@@ -4,19 +4,21 @@ Electrostatics
 ==============
 
 The Coulomb (or electrostatic) interaction is defined as
-follows. For a pair of particles at distance :math:`r` with charges
-:math:`q_1` and :math:`q_2`, the interaction is given by
+follows. For a pair of particles at distance :math:`r` with formal charges
+:math:`z_1` and :math:`z_2`, the electric potential energy is given by
```

Comment:
I don't think that `z` is a good symbol here, as it is typically used for a related but different quantity. Also I don't think `formal charge` is a common term. What is wrong with `q`?

Reply:
* isn't `z_i` the standard notation for the charge number of ions?
* the Coulomb prefactor was missing the quantity `e^2` to be consistent with the definition of the Bjerrum length in the next paragraph

This change was discussed in https://github.com/espressomd/espresso/issues/3633#issuecomment-609725762

Reply:
>  isn't z_i the standard notation for the charge number of ions?

Yes, but in the formula its charges.

> the Coulomb prefactor was missing the quantity e^2 to be consistent with the definition of the Bjerrum length in the next paragraph

The Coulomb prefactor is not related to e^2 in any way.

Reply:
Also it wasn't 'discuseed' in the cited comment, Jonas just claimed something :-)

Reply:
;-) You can do the math and check. Without the adaptions the unit of energy is not correct.